applications/nrf_desktop/configuration/nrf52840dk_nrf52840/sysbuild_fast_pair.conf

```diff
@@ -1,11 +1,15 @@
 #
-# Copyright (c) 2024 Nordic Semiconductor ASA
+# Copyright (c) 2024-2025 Nordic Semiconductor ASA
 #
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 ################################################################################
 
-SB_CONFIG_BT_FAST_PAIR=y
+# The Anti Spoofing Private Key is confidential and must be handled securely.
```

Comment:
`Anti-Spoofing Private Key` ? (to align with naming from Fast Pair doc)
Also in other places